src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -146,6 +171,69 @@ public void setDictionary(Map<String, Object> dictionary) {
     this.dictionary = Collections.unmodifiableMap(dictionary);
   }
 
+  @SuppressWarnings("unchecked")
+  public void loadDictionary() {
+    // Read and set anet-dictionary
+    final File file = new File(System.getProperty("user.dir") + "/anet-dictionary.yml");
+    try (final InputStream inputStream = new FileInputStream(file)) {
+      final Map<String, Object> dictionary = yamlMapper.readValue(inputStream, Map.class);
+      final Map<String, Object> settings = (Map<String, Object>) dictionary.get("dictionary");
+      this.setDictionary(settings);
+      // Check the dictionary
+      final JsonNode jsonNodeDictionary = checkDictionary();
+      logger.info("dictionary: {}", yamlMapper.writeValueAsString(jsonNodeDictionary));
+    } catch (IOException e) {
+      logger.error("ANET dictionary [anet-dictionary.yml] not found");
```

Comment:
we should throw an error and not continue if dictionary is not found

Reply:
In fact it was already throwing an exception on startup but just on an another step .
This is why I hadn't threw any exception at that step but with new modifed method it is ok now it handles the exceptions at that step also.